readme-developer.txt

```diff
@@ -64,6 +64,26 @@ You will probably need to adjust the paths to your compiler binaries, and you sh
 You will also need libmingw32.a and libopengl32.a. Those should be included in the MinGW g++ install. If they are not in C:\Program Files\mingw64\x86_64-w64-mingw32\lib\ you will have to adjust the paths in the Code::Blocks file.
 
 
+Windows (Visual Studio):
+
+To build 64-bit Endless Sky using Microsoft's Visual Studio
+
+1) Download the 64-bit development libraries, prebuilt specifically for use with Visual Studio:
+<fix-me-url-to-dev64-vs.zip>
+
+2) Unpack the zip file's contents into the same parent folder that contains your download of the Endless Sky repository.
+
+Example:
+\dev\endless-sky\
+\dev\dev64-vs\
+
+3) Open the Visual Studio solution located at 'endless-sky\vs\2017\endless-sky.sln'
+
+4) Compile the solution in either Release or Debug mode. A Release compile will result in machine optimations which will improve in-game performance.
```

Comment:
optimations -> optimizations (oops 😇 )
Perhaps add ` compared to the Debug compile.` to the end of the sentence.